content/kubermatic/v2.22/installation/local-kkp-installation/_index.en.md

```diff
@@ -0,0 +1,83 @@
++++
+title = "Local Kubermatic Kubernetes Platform (KKP) CE Installation"
+linkTitle = "Local Installation"
+date = 2023-06-26T09:49:10+02:00
+weight = 200
+enableToc = true
++++
+
+This page will guide you through using the KKP installer local command `kubermatic-installer local`. This command simplifies and automates the installation of KKP CE using `kind` and a preconfigured KubeVirt seed.
+
+## Pre-Installation Requirements
+
+- **Operating System:** Currently, only Linux is supported.
+- **Disk Space:** A minimum of 10Gi is recommended.
+- **RAM:** At least 8Gi of RAM is needed; 16Gi is recommended for more than a single KubeVirt user cluster.
+- **`kind`** (version 0.17 or higher): Please refer to the [`kind` documentation](https://kind.sigs.k8s.io/docs/user/quick-start/) for installation instructions.
+
+{{% notice warning %}}
+This command is experimental and not intended for production setups.
+{{% /notice %}}
+
+## Installation Procedure
+
+Follow these steps to use the KKP installer local command:
+
+1. Download the installer . This is the only manual step; there's no need to prepare any configuration since the installer should automatically configure KKP.
```

Comment:
```suggestion
1. Download the installer. This is the only manual step; there's no need to prepare any configuration since the installer should automatically configure KKP.
```